File: trail1/1부터의 합/sum-from-the-one.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int ssum = 0, mark = 0;
        for (int i=1; i<=N; i++) {
            ssum += i;
            if (ssum >= N) {
                mark = i;
                break;
            }
        }
        System.out.println(mark);
    }
}

File: trail1/1부터의 합/sum-from-the-one.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;
    int ssum = 0, mark = 0;
    for (int i=1; i<=100; i++) {
        ssum += i;
        if (ssum >= N) {
            mark = i;
            break;
        }
    }
    cout << mark << " ";
    return 0;
}